create_project.sh

```diff
@@ -13,15 +13,14 @@ mv django-react-webpack-master "$mainmenuinput"
 cd "$mainmenuinput"
 
 echo ""
-echo "--------------> Creating the python virtualenv..."
+echo "--------------> Installing python and js dependencies..."
 echo ""
-python3 -m venv .
-source bin/activate
+make install_dependencies
 
 echo ""
-echo "--------------> Installing python and js dependencies..."
+echo "--------------> Activating the poetry python virtualenv..."
 echo ""
-make install_dependencies
+poetry shell
```

Comment:
I'm almost sure we need to install the dependencies after running poetry shell, if we want to use their virtualenv. :thinking: